sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/sm_pipeline_with_comprehend.ipynb

```diff
@@ -0,0 +1,961 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    lambda_role = create_lambda_role("test-comprehend-lambda-role")
</pre><p>Use a more descriptive role name to minimize the chance of conflicts with an existing role? For S3 prefixes, we usually start with <code>DEMO</code>.  If you want to ensure uniqueness, you could also use a timestamp suffix.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2979/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>